target/linux/apm821xx/image/nand.mk

```diff
@@ -88,6 +89,9 @@ endef
 TARGET_DEVICES += netgear_wndap660
 
 define Device/netgear_wndr4700
+  DEVICE_COMPAT_VERSION := 1.1
```

Comment:
Not an issue (apart from 05_fix-compat-version). I would have no problems setting this to "3.0" as well. at least it's the same version everywhere.